tests/testAutopas/tests/selectors/AutoTunerTest.cpp

```diff
@@ -33,11 +33,25 @@ TEST_F(AutoTunerTest, testAllConfigurations) {
                                            autopas::DataLayoutOption(-1), autopas::Newton3Option(-1));
 
   // total number of possible configurations * number of samples + last iteration after tuning
-  // number of configs manually counted
+  // number of configs manually counted:
+  //
+  // Direct Sum:          directSum traversal with (AoS <=> SoA, newton3 <=> noNewton3) = 4
+  // LinkedCells:         c08 traversal       with (AoS <=> SoA, newton3 <=> noNewton3) = 4
+  //                      sliced              with (AoS <=> SoA, newton3 <=> noNewton3) = 4
+  //                      c18                 with (AoS <=> SoA, newton3 <=> noNewton3) = 4
+  //                      c01                 with (AoS <=> SoA, noNewton3)             = 2
+  //                      c01-combined-SoA    with (SoA, noNewton3)                     = 1
+  // VerletLists:         verlet-lists        with (AoS <=> SoA, newton3 <=> noNewton3) = 4
+  // VerletListsCells:    verlet-sliced       with (AoS, newton3 <=> noNewton3)         = 2
+  //                      verlet-c18          with (AoS, newton3 <=> noNewton3)         = 2
+  //                      verlet-c01          with (AoS, noNewton3)                     = 1
+  // VerletClusterLists:  verlet-clusters     with (AoS, noNewton3)                     = 1
+  //                                                                               --------
+  //                                                                                     29
```

Comment:
nice representation! Including CUDA stuff would also be helpful (-> @julian-spahl )